### ISSUE TITLE
failed to upload in TravisCI when `Appending build variables`

### ISSUE THREAD

**Issue by pchong90** [codecov/codecov-bash#161]
command used
```
 - if [ "$BUILD_TYPE" = "Debug" ] && [ "$GCC_VERSION" = 5 ]; then bash <(curl -s https://codecov.io/bash) -t token; fi;
```

output from uploader
```  _____          _
 / ____|        | |
| |     ___   __| | ___  ___ _____   __
| |    / _ \ / _` |/ _ \/ __/ _ \ \ / /
| |___| (_) | (_| |  __/ (_| (_) \ V /
 \_____\___/ \__,_|\___|\___\___/ \_/
                              Bash-8a28df4
==> Travis CI detected.
    project root: .
    Fixing merge commit SHA
    Yaml found at: .codecov.yml
==> Running gcov in . (disable via -X gcov)
==> Python coveragepy not found
==> Searching for coverage reports in:
    + .
    -> Found 1 reports
==> Detecting git/mercurial file structure
==> Appending build variables
    + TRAVIS_OS_NAME
    + TRAVIS_CMD=if
    + [
/dev/fd/63: eval: line 1247: unexpected EOF while looking for matching `]'
/dev/fd/63: eval: line 1248: syntax error: unexpected end of file
```
It seems like it has issue parsing the `TRAVID_CMD` enviroment

**Comment by kevinpapst** [comment]
Duplicate of #133 ?

**Comment by pchong90** [comment]
Yes, it is indeed a duplicate of #133.  If it has been fixed then I can close this issue as well.

**Comment by kevinpapst** [comment]
Yes, I was running into the same issue while looking for a solution.
Strange that a company who wants to earn money with their service doesn't fix such an issue after more than half a year.

**Comment by sschuberth** [comment]
@pchong90, actually, since you've confirmed this to be a duplicate of #133, you should close this one already now and subscribe to #133 instead. That said, I have #200 for review to fix #133.
